docs/design/network-policy-support.md

```diff
@@ -0,0 +1,247 @@
+
+## Multicluster Network Policy support
+
+This document describes the Network Policy use cases that we propose
+supporting for a first iteration in Submariner. In this iteration, the
+intention is to support the ingress side of existing k8s Network Policies
+as is, while in the future we could explore with the kubernetes community
+options to extend Network Policies for multi cluster scenarios.
+
+> A Kubernetes, network policy is a specification of how pods are allowed
+> to communicate with each other and other network endpoints. It is like
+> Access Control Lists using which we can specify the traffic that would
+> be allowed/denied to the PODs. Network Policies are additive, so having
+> two Network Policies that select the same Pods will result in allowing
+> both defined policies.
+
+
+> Network Policies are implemented by the networking plugin which is the
+> CNI for the K8s cluster. If the underlying network plugin does not
+> implement the Network Policies, then creating a NetworkPolicy will have
+> no effect.
+
+## Diagram
+
+The following diagram describes the example resources which will be
+analized in the use cases. It's made of two clusters, and
```

Comment:
analized -> analyzed (sic)

Also, how about "It depicts two clusters and two namespaces."